packages/analytics/analytics-utilities/src/types/explore/requests.ts

```diff
@@ -36,6 +36,7 @@ export const queryableRequestMetrics = [
   'latencies_kong_gateway_ms',
   'response_body_size',
   'request_body_size',
+  'ai_count',
```

Comment:
Shouldn't this go in the "filterable" list if we want to filter on it?

Reply:
it does doesn't it? 

```
export const filterableRequestMetrics = makeFilterable(queryableRequestMetrics)

export type FilterableRequestMetrics = typeof filterableRequestMetrics[number]

export interface RequestDimensionFilter extends Omit<ExploreFilter, 'dimension'> {
  dimension?: FilterableRequestDimensions
  field?: FilterableRequestDimensions
}

export interface RequestMetricFilter {
  type: MetricFilterTypesV2
  field: FilterableRequestMetrics
  value: number | null
}

export type RequestFilter = RequestDimensionFilter | RequestMetricFilter

```
it ends up in the `RequestMetricFilter` interface

Reply:
Ohh I see. Nevermind